miri

```diff
@@ -41,6 +47,20 @@ Pass extra flags to all cargo invocations.
 EOF
 )
 
+# Determine command.
+COMMAND="$1"
+[ $# -gt 0 ] && shift
+
+## Handle some commands early, since they should *not* alter the environment.
+case "$COMMAND" in
+many-seeds)
```

Comment:
what on earth is this syntax. I have no idea how bash works XD gonna have to trust you here

Reply:
yea I assumed that's what it is, but it looks so weird.